action.yml

```diff
@@ -7,12 +7,15 @@ branding:
 
 inputs:
   python-version: 
-    description: 'Override version of python used to run ape- default 3.8'
+    description: 'Override version of python used to run ape- default 3.10'
     required: False
-    default: '3.8'
+    default: '3.10'
```

Comment:
I think this is the only thing that makes it a breaking change

Reply:
ah ok